classes/checkout/CheckoutDeliveryStep.php

```diff
@@ -180,9 +180,9 @@ public function render(array $extraParams = array())
                     'allowed' => $this->isGiftAllowed(),
                     'isGift' => $this->getCheckoutSession()->getGift()['isGift'],
                     'label' => $this->getTranslator()->trans(
-                        'I would like my order to be gift wrapped'.$this->getGiftCostForLabel(),
-                        array(),
-                        'Checkout'
+                        'I would like my order to be gift wrapped %cost%',
```

Comment:
Oh, indeed, %cost% contains "(additional cost of %giftcost% %taxlabel%)" ([Crowdin link](https://crowdin.net/translate/prestashop-official/33242/en-fr#5719742)), not just the number :)
Ok then! Go ahead ;)